### PR TITLE
Finished Main page and About Us page.

### PR DESCRIPTION
**Pull request by dragosdinescu**

### CODE REVIEW

**Review by rmoldo — APPROVED** [review]

File: src/main/webapp/WEB-INF/pages/hero-section.jsp [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+ <link rel="canonical" href="https://getbootstrap.com/docs/4.3/examples/starter-template/">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+ <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay" crossorigin="anonymous">
+ <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!-- Bootstrap core CSS -->
+ <link href="/docs/4.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
+
+    <style>
```
Comment (ghost) [inline src/main/webapp/WEB-INF/pages/hero-section.jsp:26]: Can't be the style be in a separate file?

File: src/main/webapp/about.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+and open the template in the editor.
+-->
+<html>
+    <head>
+        <title>About Us</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <style>
+            
+            .wrapper{
```
Comment (ghost) [inline src/main/webapp/about.html:14]: Same here. Wouldn't it be cleaner with style separately? idk exactly how jsp works when it comes to this

**Review by ghost — APPROVED** [review]

**Review by Cloudy1233 — APPROVED** [review]

File: src/main/webapp/WEB-INF/tags/pageTemplate.tag [hunk 3]
```diff
@@ -20,6 +20,7 @@
     </head>
     <body>
         <jsp:include page="/WEB-INF/pages/navbar.jsp" />
+        <jsp:include page="/WEB-INF/pages/hero-section.jsp" />
```
Comment (rmoldo) [inline src/main/webapp/WEB-INF/tags/pageTemplate.tag:23]: Remove this line from the template page

**Comment by rmoldo** [comment]
As @tallarry pointed out please separate the css from the jsp. Also remove the jsp:include from the pageTemplate.jsp

**Pull request closed without merging**